core/model/src/main/kotlin/com/merxury/blocker/core/model/data/InstalledApp.kt

```diff
@@ -32,4 +33,18 @@ data class InstalledApp(
     val isEnabled: Boolean = true,
     val isSystem: Boolean = false,
     val label: String = "",
-)
+) {
+    @TestOnly
+    fun toAppItem() = AppItem(
```

Comment:
Unused function

Reply:
It's used.
![Uploading Screenshot 2024-05-10 at 13.36.23.png…]()